Packs/RiskIQDigitalFootprint/README.md

```diff
@@ -0,0 +1,30 @@
+# README #
+
+This README would normally document whatever steps are necessary to get your application up and running.
+
+### What is this repository for? ###
+
+* Quick summary
+* Version
+* [Learn Markdown](https://bitbucket.org/tutorials/markdowndemo)
+
+### How do I get set up? ###
+
+* Summary of set up
+* Configuration
+* Dependencies
+* Database configuration
+* How to run tests
+* Deployment instructions
+
+### Contribution guidelines ###
+
+* Writing tests
+* Code review
+* Other guidelines
+
+### Who do I talk to? ###
+
+* Repo owner or admin
+* Other community or team contact
+
```

Comment:
Need to replace the default README with information relevant to your pack or remove it. You can read [here](https://xsoar.pan.dev/docs/integrations/pack-docs#pack-readme) about documenting your pack.

Reply:
@avidan-H Sure, we will make this change.